spec/features/ticket_purchases_spec.rb

```diff
@@ -26,26 +26,20 @@
         fill_in "tickets__#{ticket.id}", with: '2'
         expect(current_path).to eq(conference_tickets_path(conference.short_title))
 
-        click_button 'Support'
+        click_button 'Continue'
```

Comment:
Are there feature tests for failed cases? Like when the payment doesn't go through, for example?